showcases/patternhub/pages/foundations/test-table/index.tsx

```diff
@@ -1,6 +1,7 @@
 import DefaultPage from '../../../components/default-page';
 import { DBInfotext } from '../../../../../output/react/src';
-import { testTableData } from '../../../data/testing-table'; // This file will be generated at runtime
+import { testTableData } from '../../../data/testing-table';
```

Comment:
```suggestion
import { testTableData } from '../../../data/testing-table'; // This file will be generated at runtime
```

Doesn't this file get generated at runtime anymore?